manifests/bundle-webhook/02_role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: rulpak-core-admin
+  namespace: rukpak-system
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
```

Comment:
Should we tone down these permissions down to what we just need to have Bundle validation? This could also be tackled in a follow-up.